### PR TITLE
cost/azure/hybrid_use_benefit_sql: Added subscription filter option and ability to specify Azure API endpoint

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description

cost/azure/hybrid_use_benefit_sql: Added subscription filter option and ability to specify Azure API endpoint

### CODE REVIEW

File: cost/azure/hybrid_use_benefit_sql/ahub_sql.pt [hunk 1]
```diff
@@ -29,6 +29,19 @@ parameter "param_email" do
...
+  default "management.azure.com"
+end
+
+parameter "param_subscription_whitelist" do
+  label "Subscription Whitelist"
+  type "list"
+  description "Whitelisted Subscriptions, if empty, all subscriptions will be checked"
+end
+
+
```
Comment (rshade) [inline cost/azure/hybrid_use_benefit_sql/ahub_sql.pt:44]: extra space here also

**Review by rshade — CHANGES_REQUESTED** [review]

File: cost/azure/hybrid_use_benefit_sql/ahub_sql.pt [hunk 2]
```diff
@@ -29,6 +29,19 @@ parameter "param_email" do
   label "Email addresses of the recipients you wish to notify"
 end
 
+parameter "param_azure_endpoint" do
+  type "string"
+  label "Azure Endpoint"
+  default "management.azure.com"
```
Comment (rshade) [inline cost/azure/hybrid_use_benefit_sql/ahub_sql.pt:35]: add allowed_values from #810 here.

**Review by rshade — CHANGES_REQUESTED** [review]

File: cost/azure/hybrid_use_benefit_sql/ahub_sql.pt (hunk 1 above)
Comment (XOmniverse) [reply cost/azure/hybrid_use_benefit_sql/ahub_sql.pt]: Done

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**